Installing for node does not work directly

Some dependencies are missing if you want to develop a CLI app with cozy-client-js

```
$ yarn add cozy-client-js
$ node
> require('cozy-client-js')
module.js:491
    throw err;
    ^

Error: Cannot find module 'isomorphic-fetch'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:55:19)
    at __webpack_require__ (/Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:20:30)
    at Object.<anonymous> (/Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:47:2)
    at __webpack_require__ (/Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:20:30)
    at /Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:40:18
    at Object.<anonymous> (/Users/pbrowne/code/cozy/cozy-match-operations-bills/node_modules/cozy-client-js/dist/cozy-client.node.js:43:10)
```

Here's how to add them : 

```
$ yarn add isomorphic-fetch core-js regenerator-runtime btoa
```

--- a/docs/intro.md
+++ b/docs/intro.md
@@ -1,3 +1,5 @@
+{% raw %}
+
 # Introduction to cozy-client-js
 
 ## Reminder about cozy architectures
@@ -20,15 +22,17 @@ If you already have an application using `cozy-browser-sdk`, you can see what wi
 
 ## Include the library in your application
 
-You can `import`/`require` cozy-client-js using npm & webpack
-**TODO** test if `cozy-client-js` compatible with browserify, rollup, ect.
+You can `import`/`require` cozy-client-js using npm & webpack.
 
-You can also copy-paste the `dist/cozy-client.js` bundle file into your application, and include it in your application `index.html` with  `<script src="./cozy-client.js">`.
+You can also copy-paste the `dist/cozy-client.js` bundle file into your application, and include it in your application `index.html` with `<script src="./cozy-client.js">`.
 
 If you are developing a client-side app for Cozy V3, you can import the lib directly from the stack, by using [`{{.CozyClientJS}}`](https://cozy.github.io/cozy-stack/client-app-dev.html#good-practices-for-your-application).
 
-**polyfills** : cozy-client-js uses [fetch](https://fetch.spec.whatwg.org/) and bundle all necessary polyfills (promise, fetch, reGenerator). **TODO:** We should move the polyfills to a separate bundle and require them only if needed.
+If you are developing a nodejs app, you will need some polyfills. You can add them to your project with this command:
 
+```sh
+$ yarn add isomorphic-fetch core-js regenerator-runtime btoa
+```
 
 ## Doctypes & Permissions
 
@@ -137,3 +141,4 @@ This is the end of what we already have implemented, if you want to do something
 
 Feel free to open an issue if you see something missing, or if you disagree with the API design !
 
+{% endraw %}

